Source/JavaScriptCore/runtime/JSPromiseConstructor.cpp

```diff
@@ -52,7 +57,7 @@ const ClassInfo JSPromiseConstructor::s_info = { "Function"_s, &Base::s_info, &p
 @begin promiseConstructorTable
   resolve         promiseConstructorFuncResolve        DontEnum|Function 1 PromiseConstructorResolveIntrinsic
   reject          promiseConstructorFuncReject         DontEnum|Function 1 PromiseConstructorRejectIntrinsic
-  race            JSBuiltin                            DontEnum|Function 1
+  race            promiseConstructorFuncRace                            DontEnum|Function 1
```

Comment:
Let's remove spaces before `DontEnum|Function 1` to align to the other definitions.